app/scripts/main.js

```diff
@@ -13,6 +13,7 @@ function Root() {
   return <p>Hello from Starter</p>;
 }
 
-const container = document.getElementById('app-container');
-const root = createRoot(container);
-root.render(<Root />);
+// React legacy API is used because react-helmet is not compatible with the
+// new one introduced in React v18. See:
+// https://github.com/nfl/react-helmet/issues/669#issuecomment-1134228545
+render(<Root />, document.querySelector('#app-container'));
```

Comment:
```suggestion
// React legacy API is used because there are still incompatibilities with some
// commonly used modules. An example would be react-helmet:
// (https://github.com/nfl/react-helmet/issues/669#issuecomment-1134228545).
render(<Root />, document.querySelector('#app-container'));

// If you don't need to use this module, use the new api:
// import { createRoot } from 'react-dom/client';
// const root = createRoot(document.querySelector('#app-container'));
// root.render(<Root />);
```

Since react-helmet is not a direct dependency of project seed I recommend rewriting the comment to be more generic.

Reply:
@danielfdsilva updated, thank you